packages/amplication-client/src/Entity/SchemaField.tsx

```diff
@@ -36,26 +36,33 @@ export const SchemaField = ({
   const fieldName = `properties.${propertyName}`;
   const label = propertySchema.title || capitalCase(propertyName);
 
-  const enumOptions = useMemo((): OptionItem[] | null => {
-    if (propertySchema.enum) {
-      return (propertySchema.enum as string[]).map((item) => {
-        const labelByItem: { label: string; value: string } =
-          ENTITY_FIELD_ENUM_MAPPER[fieldDataType][propertyName].find(
-            (option) => option.value === item
-          );
+  const enumOptions = useCallback(
```

Comment:
As the `propertySchema.enum`, can we keep the `useMemo`?

Reply:
Nevermind, as suggested I am going with the arbitrary array of strings approach for now.